spec/features/form_sections/teacher_training_data/placements/progress_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "completing the placements section", feature_trainee_placement: true do
+  background do
+    given_i_am_authenticated
+    given_a_trainee_exists(training_route: valid_training_routes.sample)
+  end
+
+  scenario "renders a 'incomplete' status when placement details are not provided" do
+    when_i_am_on_the_review_draft_page
+    then_the_placements_section_should_be(incomplete)
+  end
+
+  scenario "renders an 'in progress' status when placement details partially provided" do
+    when_i_visit_the_placement_details_page
+    and_i_do_not_have_the_placement_detail
+    and_i_continue
+    and_i_do_not_confirm_my_details
+    then_the_placements_section_should_be(in_progress)
+  end
+
+  scenario "renders a completed status when placement details provided" do
```

Comment:
This ideally needs one more scenario to cater for this second form https://github.com/DFE-Digital/register-trainee-teachers/pull/3698 but will update once that's merged in